src/sentry/apidocs/hooks.py

```diff
@@ -149,6 +151,35 @@ class CustomGenerator(SchemaGenerator):
     endpoint_inspector_cls = CustomEndpointEnumerator
 
 
+def find_method_and_check_paginate(content, class_name):
+    with open(content) as file:
+        file_content = file.read()
+
+    # Pattern to find the class body, either the class body is followed by another class or it's the end of the file
+    class_pattern = (
+        r"(class {class_name}.*:(.*?)(?=^class\s.*))|(class {class_name}.*:\s(.*?)(?=\Z))"
+    )
+    class_match = re.search(class_pattern, file_content, re.DOTALL | re.MULTILINE)
+
+    if class_match:
```

Comment:
It's a bit difficult to read all the if-else cases here, could you switch it to [early return](https://medium.com/swlh/return-early-pattern-3d18a41bba8)?